src/test/java/com/gepardec/mega/rest/WorkerResourceTest.java

```diff
@@ -33,29 +34,42 @@
 @JwtSecurity(claims = {
         @Claim(key = "email", value = "test@gepardec.com")
 })
-class WorkerResourceTest {
+public class WorkerResourceTest {
```

Comment:
JUnit 5 does not require public test class and methods, make class and all methods package-private